apollo-testing-support/src/appleMain/kotlin/com/apollographql/apollo3/testing/runWithMainLoopApple.kt

```diff
@@ -20,6 +20,10 @@ import platform.darwin.dispatch_get_main_queue
 import platform.darwin.dispatch_time
 import kotlin.coroutines.CoroutineContext
 
+actual fun runTest(block: suspend () -> Unit) {
+  kotlinx.coroutines.runBlocking { block() }
```

Comment:
That doesn't work for Apple tests because they need a `CFRunLoop` to be alive. There's more info about it there: https://github.com/apollographql/apollo-android/blob/dev-3.x/design-docs/Threading.md

Overall, I don't think we need a new `runTest`. Could we just rename `runWithMainLoop` to `runTest` and implement it on JS ? (and leave JVM, Apple untouched?)